javatests/arcs/sdk/wasm/EntityClassApiTest.kt

```diff
@@ -13,12 +13,12 @@ package arcs.sdk.wasm
 
 import arcs.sdk.Singleton
 
-class EntityClassApiTest(ctor: (String) -> EntityClassApiTest_Errors) :
-    TestBase<EntityClassApiTest_Errors>(ctor) {
-    private val unused1 = Singleton(this, "data") { EntityClassApiTest_Data() }
-    private val unused2 = Singleton(this, "empty") { EntityClassApiTest_Empty() }
-
-    constructor() : this({ txt: String -> EntityClassApiTest_Errors(txt) })
+class EntityClassApiTest : TestBase<EntityClassApiTest_Errors>(
+    { txt: String -> EntityClassApiTest_Errors(txt) },
```

Comment:
You should be able to use a reference to the constructor:

```suggestion
    ::EntityClassApiTest_Errors,
```